### ISSUE TITLE
v1.5.2 always requires rest-client < v2.0

### ISSUE THREAD

**Issue by manuelmeurer** [cloudinary/cloudinary_gem#234]
I just tried to update to v1.5.2 of this gem, but it seems to require rest-client < v2.0.
[Looking at the gemspec, it should only do so for Ruby 1.9](https://github.com/cloudinary/cloudinary_gem/blob/master/cloudinary.gemspec#L32), I'm running Ruby 2.4.0 though.
Where is the `RUBY_VERSION` constant set? In my case it seems to be `nil` or maybe the comparison with a string doesn't work?

**Comment by tocker** [comment]
`RUBY_VERSION` is set in [Object](https://ruby-doc.org/core-2.4.0/Object.html#RUBY_VERSION).
You can also verify that the comparison works with the following one-liner:

```
ruby -e 'puts RUBY_VERSION; if RUBY_VERSION > "2.0"; puts ">2.0"; elsif RUBY_VERSION >= "1.9"; puts ">=1.9";else;puts "<1.9"; end'
```

If rest-client is already installed it would pick it since we didn't include a minimum version for Ruby > 2.0. Try:

```
gem list rest-client
```
If the gem is listed you may want to uninstall it.

If you are installing with bundler, check your `Gemfile.lock` file to see if any other gem is specifying rest-client as dependency with a specific version.

**Comment by manuelmeurer** [comment]
Thanks for the explanation!

The one-liner works fine:

```
$ ruby -e 'puts RUBY_VERSION; if RUBY_VERSION > "2.0"; puts ">2.0"; elsif RUBY_VERSION >= "1.9"; puts ">=1.9";else;puts "<1.9"; end'
2.4.0
>2.0
```

I uninstalled all version of `rest-client`:

```
$ gem list rest-client

*** LOCAL GEMS ***

```

In my `Gemfile.lock` I indeed see two other gems requiring `rest-client` as well, both with the contraint `~> 2.0`.

Still, when I try to run `bundle install`, it says:

```
$ bi
Fetching gem metadata from https://rubygems.org/.........
Fetching version metadata from https://rubygems.org/..
Fetching dependency metadata from https://rubygems.org/.
Resolving dependencies....................................................................................................................................................
Bundler could not find compatible versions for gem "rest-client":
  In Gemfile:
    cloudinary (~> 1.5) was resolved to 1.5.2, which depends on
      rest-client (< 2.0)

    fetchy (~> 0.1) was resolved to 0.1.0, which depends on
      rest-client (~> 2.0)
```

I also tried deleting the `Gemfile.lock`, same result.

Any idea what else could be the problem? 😕

**Comment by manuelmeurer** [comment]
Will close this since it works fine with the latest version of this gem.
